packages/py3_urllib3.rb

```diff
@@ -7,6 +7,7 @@ class Py3_urllib3 < Pip
   license 'MIT'
   compatibility 'all'
   source_url 'SKIP'
+  binary_compression 'tar.zst'
```

Comment:
this package still has `no_compile_needed` and doesn't have any binaries, so either this line should be removed or binaries should be uploaded.